tests/Feature/ResourceFactoryTest.php

```diff
@@ -649,14 +661,20 @@ public function testCreateShipmentWitCustoms()
             'price'                   => 99,
```

Comment:
I couldn't put a comment at the function title, but it says WitCustoms instead of WithCustoms.